packages/docs/design-docs/ux-writing/writing-for-mobile.mdx

```diff
@@ -0,0 +1,9 @@
+# Writing for mobile
+
+When writing for the mobile app:
+
+ - Use “tap” instead of “click” when instructing the user to interact with the UI
+ - Use "enter" instead of "type" when instructing users to complete form fields
+- Keep titles, headings and subheadings short. As a good rule of thumb, try to limit copy to 2/3rds of available space, allowing 1/3rd for copy to grow longer during localization.
+- Button copy should be no longer than 3 words
+- Consider the device — not all mobile devices are the same size
```

Comment:
```suggestion
- Use “tap” instead of “click” when instructing the user to interact with the UI
- Use "enter" instead of "type" when instructing users to complete form fields
- Keep titles, headings and subheadings short. As a good rule of thumb, try to limit copy to 2/3rds of available space, allowing 1/3rd for copy to grow longer during localization.
- Button copy should be no longer than 3 words
- Consider the device — not all mobile devices are the same size
```